docs/guidance/aws-eks-gpu-cluster.md

```diff
@@ -0,0 +1,58 @@
+# Prepare Amazon EKS cluster with GPU for KubeRay examples
```

Comment:
Could you retitle this to make it more generic? How about `Start Amazon EKS Cluster with GPUs for KubeRay`?

Reply:
https://github.com/ray-project/kuberay/pull/1181/commits/4d018b2fb5561bffb45c83eb0bcf6b7c9d1db9f5